lib/storage/bucket.js

```diff
@@ -267,6 +287,9 @@ Bucket.prototype.getFiles = function(query, callback) {
       file.metadata = item;
       return file;
     });
+    if (resp.prefixes) {
+      files.prefixes = resp.prefixes; // bad idea?
```

Comment:
What's the motivation for this? If the motivation is :+1: then I'd say it's a good idea :smile: